recipe/recipe.yaml

```diff
@@ -146,6 +131,7 @@ outputs:
         - nlohmann_json
         - openssl
         - pcre
+        - libgl-devel  # [linux]
```

Comment:
this needs to be rattler-ized. There's already an `- if: linux` further up where you could smuggle this in. ;-)

PS. numba is now 3.13 compatible

Reply:
🤦 indeed! That linter comment is very useful as I think my muscle memory is going to take a while to un-learn selectors.

Btw, thanks for the reivew! 🍰